.circleci/config.yml

```diff
@@ -488,7 +488,7 @@ jobs:
               --os-version-ids 29 \
               --locales de_DE \
               --orientations portrait \
-              --no-record-video
+              --no-record-video | true
```

Comment:
Please revert this, I have updated my PR, this change should achieve the right behaviour https://github.com/corona-warn-app/cwa-app-android/pull/2258/commits/e53c6762d6c41f540cf761620944601c29f099da

Reply:
Reverted.